templates/book/reader-files-epub.html.twig

```diff
@@ -0,0 +1,24 @@
+{% extends themedTemplate('bare.html.twig') %}
+
+{% block title %}{{ book.authors|first }} {% if book.serie is not null %}> {{ book.serie }} #{{ book.serieIndex }} {% endif %}> {{ book.title }}{% endblock %}
+{% block javascripts %}
+    {{ encore_entry_script_tags('read-ebook') }}
+    {{ parent() }}
+{% endblock %}
+
+{% block stylesheets %}
+    {{ encore_entry_link_tags('read-ebook') }}
+    <style>
+        body
+        {
+            background-color: #000;
+            color: #fff;
+        }
```

Comment:
https://media3.giphy.com/media/7SF5scGB2AFrgsXP63/giphy.gif

Reply:
I moved that to scss, and injected a class instead. 

I also took care of changing the vue component to better support theming.